src/global.d.ts

```diff
@@ -0,0 +1,19 @@
+declare var VERSION: string
+
+declare module '*.svg' {
```

Comment:
It basically tells typescript that you can import image files. 

https://stackoverflow.com/questions/51100401/typescript-image-import

Otherwise, he will won't know what to do with this line:
```js
import unknownTokenImg from 'img/unknown-token.png'
```
We are telling that an import of a an image returns `anything`

Reply:
Understood, thanks!